database/migrations/2021_05_31_140156_create_prefectures_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreatePrefecturesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('prefectures', function (Blueprint $table) {
+            $table->id();
+            $table->integer('area_id');
```

Comment:
説明を追加しておいてもらえると嬉しいです！

```suggestion
            $table->integer('area_id')->comment('1.北海道・東北、2.関東、3.中部、4.関西、5.四国・中国、6.九州・沖縄');
```

Reply:
データを後から入れるだけではなくてここにも説明を入れるということですね？